src/style.css

```diff
@@ -1,3 +1,95 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --select-color: #00acdc;
+  --main-color: #000;
+  --font-family: roboto, sans-serif;
+  --main-text-size: 16px;
+}
+
+html,
+body {
+  height: 100%;
+}
+
 body {
```

Comment:
Try not to refer to tags directly, but do it through classes.

Reply:
Problem is fixed